services/fee-estimator/shared/dynamicFees.js

```diff
@@ -18,7 +18,7 @@ const { CacheRedis, Logger } = require('lisk-service-framework');
 
 const { calcAvgFeeByteModes, EMAcalc } = require('./utils/dynamicFees');
 const { parseInputBySchema } = require('./utils/parser');
-const { getTxnMinFee, getAllTransactionSchemasFromCache } = require('./utils/transactionsUtils');
+const { getTxnMinFee, getAllCommandsParamsSchemasFromCache } = require('./utils/transactionsUtils');
```

Comment:
Kindly rename the method to `getAllCommandsParamsSchemas` and make other changes accordingly. It can still retain its caching feature.
```suggestion
const { getTxnMinFee, getAllCommandsParamsSchemas } = require('./utils/transactionsUtils');
```